tests/test_dataset.py

```diff
@@ -1356,24 +1353,20 @@ def test_writing_subset_of_compressed_data_multi_channel() -> None:
 
     # create uncompressed dataset
     write_data1 = (np.random.rand(3, 20, 40, 60) * 255).astype(np.uint8)
-    WKDataset.create(
-        os.path.abspath("./testoutput/compressed_data"), scale=(1, 1, 1)
-    ).add_layer("color", Layer.COLOR_TYPE, num_channels=3).add_mag(
-        "1", block_len=8, file_len=8
-    ).write(
-        write_data1
-    )
+    WKDataset.create(Path("./testoutput/compressed_data"), scale=(1, 1, 1)).add_layer(
+        "color", Layer.COLOR_TYPE, num_channels=3
+    ).add_mag("1", block_len=8, file_len=8).write(write_data1)
 
     # compress data
     compress_mag_inplace(
-        os.path.abspath("./testoutput/compressed_data/"),
+        Path(os.path.abspath("./testoutput/compressed_data/")),
```

Comment:
We could get rid of the os.path here by changing this to `(TEST_OUTPUT_DIR / "compressed_data").resolve()` (assuming my suggestion from above is implemented).
Assuming that the specific symlink behavior of abspath is not needed here? Compare my notes on abpath in the description here https://github.com/scalableminds/voxelytics/pull/2175